src/lib/rest/clientSocketHttp.cpp

```diff
@@ -291,8 +292,15 @@ std::string sendHttpSocket
   }
 
   // ----- Accept
-  headers = curl_slist_append(headers, "Accept: application/xml, application/json");
-  outgoingMsgSize += 41; // from "Accept: application/xml, application/json"
+  std::string acceptedFormats = "application/xml, application/json";
```

Comment:
...and I think the Accept header is set based on the "notify preferences", right?